Formula/podman.rb

```diff
@@ -14,6 +15,7 @@ class Podman < Formula
 
   depends_on "go" => :build
   depends_on "go-md2man" => :build
+  depends_on "qemu" if Hardware::CPU.intel?
```

Comment:
Can we add a (small) test for any functionality that uses `qemu`?

Reply:
Most functionality that uses qemu requires a vm (large). I can add a test that will fail, something like 
`podman machine init --image-path /bogus` that will fail with `Error: open /bogus: no such file or directory` with qemu and without qemu will fail with `Error: exec: "qemu-system-x86_64": executable file not found in $PATH`, if that sounds okay